client/src/featureform/get_local.py

```diff
@@ -371,3 +372,11 @@ def get_variant(resource_type, name, variant):
 def format_resource_list(res_list, name_col="name", var_col="variant"):
     formatted_res = [{ "name": r[name_col], "variant": r[var_col]} for r in res_list]
     return formatted_res
+
+def get_model_info_local(name) -> Model:
+    model = get_resource("model", name)
+
+    format_rows("MODEL NAME: ", model["name"])
```

Comment:
Always thought it was a bad idea to make the get call also print stuff, seems like it'd be annoying if I was just using FF normally

Reply:
No need to change in this PR

Reply:
I've been discussing this with @sdreyer here and there, and I agree it would be nice to separate out concerns here, as well as introduce more formatting options. But agreed: well out of scope for this PR, as well as the RFC.